modules/cluster-logging-exported-fields-top-level-fields.adoc

```diff
@@ -10,38 +9,52 @@
 
 The top level fields may be present in every record.
 
-=== message
-
+== message
 
 Original log entry text, UTF-8 encoded This field may be absent or empty if a non-empty `structured` field is present. See the description of `structured` for more.
 
-=== structured
+[horizontal]
+Data type:: text
+Example value:: `HAPPY`
 
+== structured
 
 Original log entry as a structured object. This field may be present if the forwarder was configured to parse structured JSON logs. If the original log entry was a valid structured log, this field will contain an equivalent JSON structure. Otherwise this field will be empty or absent, and the `message` field will contain the original log message. The `structured` field has whatever sub-fields are included in the log message, there are no restrictions defined here.
```

Comment:
Same as above - article missing? "The original log entry". All other items below have one.

Subfields should be spelled as one word: https://ahdictionary.com/word/search.html?q=subfield

The whole sentence seems a bit colloquial: The `structured` field has whatever sub-fields are included in the log message, there are no restrictions defined here.

Maybe it could be cut to: The `structured` field can have any subfields that are included in the log message (or from the log message), WDYT?

Reply:
Updated. Thanks.